config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -0,0 +1,116 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:55680
+      http:
+        endpoint: 0.0.0.0:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  awsecscontainermetrics:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+  filter:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - ecs.task.memory.reserved
+          - ecs.task.memory.utilized
+          - ecs.task.cpu.reserved
+          - ecs.task.cpu.utilized
+          - ecs.task.network.rate.rx
+          - ecs.task.network.rate.tx
+          - ecs.task.storage.read_bytes
+          - ecs.task.storage.write_bytes
+  metricstransform:
+    transforms:
+      - metric_name: ecs.task.memory.utilized
+        action: update
+        new_name: MemoryUtilized
+      - metric_name: ecs.task.memory.reserved
+        action: update
+        new_name: MemoryReserved
+      - metric_name: ecs.task.cpu.utilized
+        action: update
+        new_name: CpuUtilized
+      - metric_name: ecs.task.cpu.reserved
+        action: update
+        new_name: CpuReserved
+      - metric_name: ecs.task.network.rate.rx
+        action: update
+        new_name: NetworkRxBytes
+      - metric_name: ecs.task.network.rate.tx
+        action: update
+        new_name: NetworkTxBytes
+      - metric_name: ecs.task.storage.read_bytes
+        action: update
+        new_name: StorageReadBytes
+      - metric_name: ecs.task.storage.write_bytes
+        action: update
+        new_name: StorageWriteBytes
+  resource:
+    attributes:
+      - key: ClusterName
+        from_attribute: aws.ecs.cluster.name
+        action: insert
+      - key: aws.ecs.cluster.name
+        action: delete
+      - key: ServiceName
+        from_attribute: aws.ecs.service.name
+        action: insert
+      - key: aws.ecs.service.name
+        action: delete
+      - key: TaskId
+        from_attribute: aws.ecs.task.id
+        action: insert
+      - key: aws.ecs.task.id
+        action: delete
+      - key: TaskDefinitionFamily
+        from_attribute: aws.ecs.task.family
+        action: insert
+      - key: aws.ecs.task.family
+        action: delete
+exporters:
+  awsxray:
+    region: us-west-2
+  awsemf/application:
+    region: us-west-2
+    log_group_name: 'aws/ecs/containerinsights/{ClusterName}/application/metrics'
+    log_stream_name: '{TaskId}'
+  awsemf/performance:
+    namespace: ECS/ContainerInsights
```

Comment:
We can't hard code the region. It needs to be dynamic so that this config can work in any region, and any partition.

@mxiamxia @hossain-rayhan if we don't explicitly state a region in the config, the exporters will autodetect the current region right?

Reply:
I think so. @mxiamxia can confirm.

Reply:
I deleted and tested it, it works.